docs/model.md

```diff
@@ -5,27 +5,30 @@ title: EpiJS Module - Model
 
 ### Table of Contents
 
-*   [Model][1]
-    *   [Parameters][2]
-    *   [get_data][3]
-        *   [Examples][11]
+
+*   [Model][2]
+    *   [Parameters][3]
+    *   [Examples][4]
+    *   [get_data][5]
+        *   [Parameters][6]
+        *   [Examples][7]
```

Comment:
Why are we changing this?

Reply:
When we regenerated the documentation for new URLs, this might have been changed.